scoring/update.html

```diff
@@ -1,5 +1,80 @@
 {% extends "_update.html" %}
 
-{% block valid_token_regex %}
-var valid_token_regex = /^[GOPY]*$/;
+{% macro input_highest(x, y, name) %}
+    <foreignObject x="{{ x + 5 }}" y="{{ y }}" width="95" height="30">
+        <input
+            class="highest tokens"
+            placeholder="Highest"
+            type="text"
+            id="district_{{ name }}_highest"
+            name="district_{{ name }}_highest"
+            value="{{ request.form.get('district_{}_highest'.format(name)) | empty_if_none }}"
+            onkeyup="token_input_change(this);"
+            pattern="^[GOPY]?$"
+        />
+    </foreignObject>
+{% endmacro %}
+
+{% macro input_pallets(x, y, name, colour_symbol) %}
```

Comment:
Do you mean in the `input` fields? So having the letters the user is typing being in the relevant colours?

If so, I'm not sure that that would actually help -- the score-sheets which are the reference input will be in either black or blue pen and the user _does not need to know_ what the letters stand for.

Reply:
Here's my proposal, add a glow to the letters, similar to the zone numbers
![image](https://github.com/user-attachments/assets/35df18b9-96ce-4fda-9a94-dedcb9b3b916)

Reply:
Oh, I see. (I had also forgotten that we'd made that layout change :facepalm:). Yeah, I can definitely see this being useful in the virtual competition where we're going straight in rather than copying from the paper sheets (which as mentioned are greyscale).

I'll have a look at how easy this is to do.

Reply:
Done in https://github.com/srobo/sr2025-comp/pull/1/commits/f473854eff8e228867f7b24a28ab204fa949a064